consistency-tests/__tests__/clickables.test.tsx

```diff
@@ -38,14 +38,13 @@ const IconButtonWrapper = (props: any) => (
 );
 
 describe.each`
-    Component            | name             | role
-    ${ActionItem}        | ${"ActionItem"}  | ${"menuitem"}
-    ${Button}            | ${"Button"}      | ${"button"}
-    ${ClickableWrapper}  | ${"Clickable"}   | ${"link"}
-    ${CompactCell}       | ${"CompactCell"} | ${"link"}
-    ${DetailCell}        | ${"DetailCell"}  | ${"link"}
-    ${IconButtonWrapper} | ${"IconButton"}  | ${"link"}
```

Comment:
**note:** The relevant tests have now been moved to `icon-button.test.tsx`. Note that I removed the `window.open` test as we no longer require to test this with `IconButton` because we let the browser handle this for us, so there's no need to test implementation details in that case from now on.

Reply:
ah I meant about the `IconButtonWrapper` part :). All the other tests for other clickable elements will remain here.

You can see the tests copied to icon-button here: https://github.com/Khan/wonder-blocks/pull/2090/files/753c9ebc30f9ea81fb1494644abd5d0e434e8f74#diff-281fe82fd57699c210462f743c18eb5434a47b22ef793de121d7151dce6f5bdeR165

Reply:
I see, thanks!